packages/web/src/scss/components/CheckboxField/_CheckboxField.scss

```diff
@@ -52,10 +55,12 @@
     @include form-fields-tools.message();
 }
 
-.CheckboxField--error .CheckboxField__message {
-    color: form-fields-theme.$message-color-error;
+.CheckboxField__helperText {
+    @include form-fields-tools.helper-text();
 }
 
+@include form-fields-tools.box-field-variants($_field-name);
```

Comment:
I think we should rename the mixin. Box is for text... form fields. Checkbox and Radio are inline. It should be something reusable in both types.